firefox-suggest-history-vs-offline.toml

```
@@ -1,5 +1,43 @@
-[metrics]
+[experiment]
+Segments = ['urlbar_exposed', 'urlbar_users']
```

Looks like there is a typo here
```suggestion
segments = ['urlbar_exposed', 'urlbar_users']
```